src/portal.rs

```diff
@@ -40,12 +37,11 @@ api_planning! {
   }
 }
 
-const CONTEXT_NOT_FOUND_ERROR_MESSAGE: &str =
-  "failed to find `PortalCtx`, make sure you are using `<PortalProvider />` \
+const CONTEXT_NOT_FOUND_ERROR_MESSAGE: &str = "failed to find `PortalCtx`, make sure you are using `<PortalProvider />` \
    somewhere near the root of the app";
 
 #[derive(Clone)]
-struct PortalCtx(StoredValue<Vec<(TypeId, RwSignal<Option<Children>>)>>);
+struct PortalCtx(StoredValue<Vec<(TypeId, RwSignal<Option<ChildrenFn>>)>>);
```

Comment:
This was a change intentionally made. It would be better if we can use `Children` instead of `ChildrenFn`. What's the reason we need `ChildrenFn` again?

Reply:
It doesn't compile with `Children`, likely due to the changes in the ownership system, but I must admit I didn't investigate further.

Reply:
Ahh, got it. So one of the reasons why we changed it to `Children` was because a lot of times, in custom components, we take in `Children` as a prop, and pass that onto the portal component. If we can retain that, that would be amazing

Reply:
`Children` can only be called once, because it's type alias is `FnOnce() -> View`, whereas `ChildrenFn` is `Fn() -> View`. This means that if you use `Children`, you can only ever call the children function exactly once. If you need children to render more than once, such as when moving into an `Fn()` for `DynChild`, then you must use `ChildrenFn`, because it lets you call the children

Reply:
Can we perhaps use `FnOnce() + Clone` then?

I remember having significant issues with using `ChildrenFn` instead of `Children`, which is why the change was made.

Alternatively, can we perhaps add an example / test for making a custom component on top of the portal so that we can make sure it compiles?

Reply:
I haven't yet reviewed the PR, but it should be possible to use `Children` without requiring `Clone`...that's what the `Option<Children>` is for...when the children changes, we can `children.take()` and then render the `FnOnce() -> View`.